dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/passwordreset/PasswordResetInitResponse.java

```diff
@@ -21,24 +21,6 @@ public PasswordResetInitResponse(boolean emailFound) {
     public PasswordResetInitResponse(boolean emailFound, PasswordResetData passwordResetData) {
```

Comment:
Now that I'm thinking, I'm not sure if enforcing resetUrl in constructor wouldn't be better.